ext/twig/php_twig.h

```diff
@@ -2,34 +2,54 @@
    +----------------------------------------------------------------------+
    | Twig Extension                                                       |
    +----------------------------------------------------------------------+
-   | Copyright (c) 2011 Derick Rethans                                    |
+   | Copyright (c) 2011 Sensiolabs                                        |
    +----------------------------------------------------------------------+
    | Redistribution and use in source and binary forms, with or without   |
    | modification, are permitted provided that the conditions mentioned   |
    | in the accompanying LICENSE file are met (BSD-3-Clause).             |
    +----------------------------------------------------------------------+
    | Author: Derick Rethans <derick@derickrethans.nl>                     |
+   |         Julien PAULI <jpauli@php.net>                                |
    +----------------------------------------------------------------------+
  */
 
 #ifndef PHP_TWIG_H
 #define PHP_TWIG_H
 
-#define PHP_TWIG_VERSION "2.0.0-DEV"
+#define PHP_TWIG_VERSION "2.0.0"
```

Comment:
this change is wrong. It should keep the ``-DEV`` suffix outside of releases, to be consistent with the way the Twig version number is handled